mne/utils/check.py

```diff
@@ -330,14 +330,29 @@ def _soft_import(name, purpose, strict=True):
         functionality the package provides to MNE-Python.
     strict : bool
         Whether to raise an error if module import fails.
+    pip_name : str
+        The PyPI package name if it does not match the module name.
     """
+    # If PyPI package name matches import namespace
+    # pip_name = name if pip_name is None else pip_name
+    pip_name = dict(
+        sklearn='scikit-learn',
+        EDFlib='EDFlib-Python',
+        mne_bids='mne-bids',
+        mne_nirs='mne-nirs',
+        mne_features='mne-features',
+        mne_qt_browser='mne-qt-browser',
+        mne_connectivity='mne-connectivity',
+        pyvista='pyvistaqt').get(name, name)
+
     try:
         mod = import_module(name)
         return mod
     except (ImportError, ModuleNotFoundError):
         if strict:
             raise RuntimeError(f'For {purpose} to work, the {name} module is '
-                               'needed, but it could not be imported.')
+                               'needed, but it could not be imported. '
+                               f'pip install {pip_name} and try again.')
```

Comment:
we want to provide both `conda` and `pip` commands here I think.
see https://github.com/mne-tools/mne-python/issues/10562#issuecomment-1109739029

I'd also suggest inserting newline characters (or equivalently, making the error message a mulitline string) so that the commands show up alone on a line, for easier select-copy-paste.

Reply:
Sorry - I made your requested changes on my fork instead of committing your suggestions here. I will just commit suggestions moving forward:

I revised the code in `_soft_import` to insert new line characters, and also indent each new line so that the error message lines are aligned - if the inner function I defined for this purpose isn't advisable, just let me know. 

I also added conda instructions.

Thanks Daniel